vendor.conf

```diff
@@ -0,0 +1,37 @@
+github.com/Microsoft/go-winio c40bf24f405ab3cc8e1383542d474e813332de6d
+github.com/Sirupsen/logrus 3ec0642a7fb6488f65b06f9040adc67e3990296a
+github.com/urfave/cli c31a7975863e7810c92e2e288a9ab074f9a88f29
+
+# coreos/etcd depends on the dependencies listed below
+# their import version should be the same as that required
+# by the corresponding version of coreos/etcd
+github.com/coreos/etcd 6081a29c13c380f1e7d85240f7927e422b0cfa51
+# golang.org/x/net/context 7dbad50ab5b31073856416cdcfeb2796d682f844
```

Comment:
Why this is commented? I can find `vendor/golang.org/x/net/context/context.go` below. Should be removed if it's not needed.